### ISSUE TITLE
How to run your code?

### ISSUE THREAD

**Issue by bitlater** [UNIR-Anonymous/UNIR#1]
Hello! Thank you for such a good work! I wonder how one can run your code, for instance on MNIST or CelebA datasets? In particular (I may be wrong) it seems that module 'sacred' is absent and at the same time as I understand it is crutial. Thank you.

**Comment by pajotarthur** [comment]
Hi,
Sacred is available at : https://github.com/IDSIA/sacred

Yon can also do : pip install -r requirements.txt on the following file.

numpy
torch
torchvision
git+git://github.com/IDSIA/sacred.git#egg=sacred
pyyaml
prettytable
pytorch-ignite>=0.2.0
imageio

**Comment by bitlater** [comment]
Thank you, I solve this issue, but now I ran into another problem: module 'unsup_it' not found.

**Comment by Wenchao-Du** [comment]
Hi, the code you released could not run, and now, we meet a problem about the loss of  mongo.json file, i could not find the file in your code package, thank you. @pajotarthur

**Comment by pajotarthur** [comment]
> Hi, the code you released could not run, and now, we meet a problem about the loss of mongo.json file, i could not find the file in your code package, thank you. @pajotarthur

Hi,
You should just set the option use_mongo to false in main.py. 
For running the code, one must replace the occurrence of unsup_it to unir (the name of the main folder). I tried to push but unfortunately lost access to the anonymous account.

**Comment by Wenchao-Du** [comment]
> > Hi, the code you released could not run, and now, we meet a problem about the loss of mongo.json file, i could not find the file in your code package, thank you. @pajotarthur
> 
> Hi,
> You should just set the option use_mongo to false in main.py.
> For running the code, one must replace the occurrence of unsup_it to unir (the name of the main folder). I tried to push but unfortunately lost access to the anonymous account.

@pajotarthur ,yes, i have replace the path unsup_it with the main folder, i would try it again obeying your advice. thank you.
